src/main/java/com/googlecode/loosejar/output/CSVFormatSummarizer.java

```diff
@@ -0,0 +1,52 @@
+package com.googlecode.loosejar.output;
+
+import java.util.List;
+import java.util.Map;
+
+import com.googlecode.loosejar.ClassLoaderAnalyzer;
+import com.googlecode.loosejar.JarArchive;
+
+public class CSVFormatSummarizer implements Summarizer {
```

Comment:
The idea was to avoid any dependency on library for following reasons

1.  out of the 5 columns in csv (results) only 2 were character based out of which one was jar path and other was the classloader name. I do not see both of them having a comma within them which would spoil the csv
2. added dependency into jvm
3. using a library providing CSV RFC would be kind of a overkill here for just 2 columns

I opened the CSV using MS Office 2016 and couldnt see any issues, from my understanding quotes provide the isolation when there is a comma expected within the column itself. However on a safer side we can keep the columns quoted.